### ISSUE TITLE
[MAS4.2.1] [Screen Reader-Restart conversation menu] When the user selected the restart conversation narrator focus remains on the same and reading as “Document “which is not descriptive 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1752]
**Actual Result:**
When the user selected the restart conversation narrator focus remains on the same and reading as “Document” which is not descriptive 

**Expected Result:**
 Screen reader should narrate the proper info when we restart conversation selected 
When the restart conversation selected focus should be on the Type your message text box and should read the Conversation restarted type your message.

**Note:** Issue is repro’ing with NVDA 

**Repro Steps:**
Turn on the Narrator 
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened
3.Navigate to the open bot and select it  
4. Dialog gest opened and connect the open bot  
5. Live chat screen gets opened 
6.Navigate to the Restart conversation and select it 
7. Observe the issue. 

**User Impact:**
Users who rely on screen reader will be affected to navigate the control if screen reader is reading in appropriate information for restart conversation item screen reader user will be affected.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator
Screen reader: Narrator 

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}](url)
[MAS4.2.1_Restart conversation.zip](https://github.com/microsoft/BotFramework-Emulator/files/3519857/MAS4.2.1_Restart.conversation.zip)
![MAS4 2 1_Restart](https://user-images.githubusercontent.com/53518994/63337170-1e0cac80-c35e-11e9-94c4-d8158b6cfc11.png)

**Comment by Kumar2608** [comment]
Verified. Issue is working fine in NVDA. Hence Closing the issue
NVDA Version: 2019.1.1
